src/components/App.js

```diff
@@ -11,7 +11,7 @@ const App = props => {
       </BrowserRouter>
       <Orbit />
     </div>
-  );
-};
+  )
+}
 
-export default App;
+export default App
```

Comment:
y'all've been doing the semi-colon dance a lot. 
Please decide as a group to not use semi-colons, and it'll save some confusion.

Reply:
decided to not use semi-colons.  Everyone has updated their prettier settings to not add them.

Reply:
:D